dataprofiler/data_readers/text_data.py

```diff
@@ -45,6 +51,7 @@ def __init__(self, input_file_path=None, data=None, options=None):
             raise ValueError("Input data type is not string.")
 
         options = self._check_and_return_options(options)
+        options = cast(Dict, options)
```

Comment:
any chance we can get rid of this `cast`?

Reply:
Looks like that cast is no longer needed. Removed it.

Reply:
awesome